linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -129,6 +129,12 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_LINUX_07_010: [** `insert_idx` and `consume_idx` for the task array shall be initialized to 0. **]**
 
+**S_R_S_THREADPOOL_LINUX_11_001: [** `threadpool_create` shall initialize internal threapool data items **]**
```

Comment:
>threadpool_create` shall initialize internal threapool data items [](http://example.com/codeflow?start=38&length=65)

This spec is not really testable. I would not know what to write in a test for it, so I think it should be removed.

Reply:
We are testing it in `threadpool_linux_ut.c` at line 465.